docs/install/operator/knative-with-operator-cli.md

```diff
@@ -12,7 +12,7 @@ Before you install the Knative Operator CLI Plugin, first install the [Knative C
 
 === "MacOS"
 
-    1. Download the binary `kn-operator-darwin-amd64` for your system from the [release page](https://github.com/knative-sandbox/kn-plugin-operator/releases/tag/knative-v1.6.0).
+    1. Download the binary `kn-operator-darwin-amd64` for your system from the [release page](https://github.com/knative-sandbox/kn-plugin-operator/releases/tag/knative-v1.6.2).
```

Comment:
Might be worth just linking to the main release page (rather than a specific version) so that this won't require an update on each new release

Reply:
well we have different version . for example, if people click on the knative-1.6 docs, I hope that it will link them to the 1.6 version instead of the latest released version, which can be 1.7, 1.8, or something.